src/main/java/org/apache/sling/jcr/resource/internal/helper/jcr/JcrNodeResourceMetadata.java

```diff
@@ -35,6 +35,8 @@
 import javax.jcr.Property;
 import javax.jcr.RepositoryException;
 import javax.jcr.ValueFormatException;
+import org.apache.jackrabbit.JcrConstants;
+import static org.apache.jackrabbit.JcrConstants.NT_FILE;
```

Comment:
Please clean up imports and only use constants from https://docs.adobe.com/docs/en/spec/javax.jcr/javadocs/jcr-2.0/constant-values.html